Cargo.toml

```diff
@@ -8,3 +8,4 @@ edition = "2018"
 anyhow = "1.0"
 lz-fear = "0.1"
 clap = "3.0.0-beta.2"
+libremarkable = "0.4.3"
```

Comment:
This pulls in a lot of unnecessary dependencies, I already opened a PR (linked earlier) to reduce that somewhat. Doesn't seem to have an effect on binary size, though.